config/roguelike_dungeons/settings/xloot_builtin_override.json

```diff
@@ -0,0 +1,8 @@
+// This file replaces the builtin loot with custom loot
+// This works by replacing the ID for the builtin loot with a new definition.
```

Comment:
Those comments should go in the readme file. As you can see comments are not supported in json files.